src/hooks/use-user-name.jsx

```diff
@@ -21,13 +25,29 @@ const useUserName = () => {
 
   const updateFirstName = (newFirstName) => {
     setFirstName(newFirstName)
+    if (!newFirstName.trim()) {
```

Comment:
There are already validation functions for name fields. Wouldn't it be better to use them to have the uniform rules for all forms?